### PR TITLE
saving and experiment set up

### PR DESCRIPTION
**Pull request by davidhuizinga**

### CODE REVIEW

**Review by pepijnbruinaars — CHANGES_REQUESTED** [review]
Looks good for the most part, left a few inline comments with Milo.

On a general note, I think we should decrease the task count per gendered voice to 5 instead of 10. It seems a bit much to me, and I wouldn't voluntarily want to have 20 conversations with some random chatbot lol.

File: src/dialog_manager.py [hunk 1]
```diff
@@ -232,6 +245,65 @@ def start_dialog(self):
             print()
             self.__handle_exit()
 
+    def start_experiment(self):
+        
+        self.dialog_config["tts"] = True
+        tasks = pd.read_csv("data/experiment_data/tasks.csv", sep=";")["tasks"].tolist()
+        first_gender = input("Do you want to start with male or female? enter \"m\" or \"f\": ")
```
[the remark this answers is not in the document]
Comment (davidhuizinga) [reply src/dialog_manager.py]: I think this correctly implements a predetermined order for group A and group B:
group A gets first 5 male then 5 female
group B gets first 5 female then 5 male
so when we all interview four people we have to select the male John for 2 of them and Jane for the other 2. 
Was this what we discussed during the workgroup? I'm not sure anymore lol

**Review by Milooooo1 — APPROVED** [review]
Looks good let start the experiments :)

**Review by Milooooo1 — APPROVED** [review]
Looks good lets start experimenting :)

File: src/dialog_manager.py [hunk 2]
```diff
@@ -354,7 +433,8 @@ def __handle_inform(self, prepped_user_input) -> bool:
         # If 1 option left, suggest it, we also cannot ask the user for additional preferences
         if restaurant is not None and other_options is None:
             self.__respond(self.__get_suggestion_string(restaurant))
-            self.__prompt_additional_preferences()
+            if(self.dialog_config["experiment"] == False):
```
Comment (pepijnbruinaars) [inline src/dialog_manager.py:436]: Just a minor remark, doesn't matter that match, but it's more concise to write this as `if not self.dialog_config["experiment"]:`

**Review by pepijnbruinaars — APPROVED** [review]
Let's go!

**Pull request merged**